pkg/deploy/api/test/ok.go

```diff
@@ -160,6 +160,36 @@ func OkPodTemplate() *kapi.PodTemplateSpec {
 	}
 }
 
+func OkPodTemplateMissingImageWithOkImageChangeTrigger() *kapi.PodTemplateSpec {
```

Comment:
I have no problem for making a helper for this but it could be
```go
func OkPodTemplateMissingImage(missing ...string) *kapi.PodTemplateSpec {
    set := sets.NewString(missing...)
    template := OkPodTemplate()
    for _, c := range template.Spec.Containers {
       if set.Has(c.Name) {
            c.Image =  ""
       }
    }
    return template
}
```
Back to the caller you would need to pass ContainerNames from the ICT in this helper.